WtWSMS/history/characters/WTWSMS_enlil.txt

```diff
@@ -1075,8 +1074,7 @@
 }
 
 36069 = {
-	name = "Muritta"
-	# Muritta Silingi
+	name = "Fredeberga" # replace the name "Muritta" since it is a male name (for example St. Muritta is mentioned together with St. Eugenius in the Roman Martyrology on July 13.)
```

Comment:
```suggestion
	name = "Fredeberga" # replace the name "Muritta" since it is a male name (for example St. Muritta is mentioned together with St. Eugenius in the Roman Martyrology on July 13.)
	# Muritta Silingi
```